### PR TITLE
Prevent further level impersonation

### PR DESCRIPTION
**Pull request by sharidas**
Prevent further level of impersonation.

Signed-off-by: Sujith H <sharidasan@owncloud.com>

### CODE REVIEW

File: js/impersonate.js [hunk 1]
```diff
@@ -150,6 +150,8 @@
 					OC.dialogs.alert(t('impersonate', result.responseJSON.message), t('impersonate', "Error"));
 				} else if ((result.responseJSON.error === "cannotImpersonateAdminUser") && (result.responseJSON.message.length > 0)) {
 					OC.dialogs.alert(t('impersonate', result.responseJSON.message), t('impersonate', "Error"));
+				} else if ((result.responseJSON.error === "stopNestedImpersonation") && (result.responseJSON.message.length > 0)) {
+					OC.dialogs.alert(t('impersonate', result.responseJSON.message), t('impersonate', "Error"));
```
Comment (DeepDiver1975) [inline js/impersonate.js]: this will not work - the message cannot be extracted by gettext. 
Translate the error on server side in php.

Comment (sharidas) [reply js/impersonate.js]: @DeepDiver1975 I made the change in the server side. Let me know if you are ok with the change.

Comment (PVince81) [reply js/impersonate.js]: @sharidas then remove the `t()` call here if the message is already translated

Comment (sharidas) [reply js/impersonate.js]: Updated.

**Comment by SergioBertolinSG** [comment]
please don't merge this until I test the problem is fixed

**Comment by sharidas** [comment]
> please don't merge this until I test the problem is fixed

Sure.

**Comment by SergioBertolinSG** [comment]
OK works 👍

**Comment by sharidas** [comment]
Merge button is disabled. Kindly merge this change please. QA had verified the same.

**Pull request merged**